metricbeat/module/mongodb/collstats/_meta/docs.asciidoc

```diff
@@ -0,0 +1,13 @@
+This is the `collstats` metricset of the module mongodb.
+
+It is using the `top` adminitrative command to return usage statistics for each collection. It provides the amount of time, in microseconds, used and acount of operations for the following types:
```

Comment:
Thanks, I think this will be helpful. 

"acount" looks like a typo. Can you fix that?